### DISCUSSION TITLE
Backlink Renderer should render the markdown

### DISCUSSION THREAD

**Discussion by hgad** [GitJournal/GitJournal#743] [Feature Requests]
## Environment

**App version:** GitJournal 1.71.1+1939
**Device information:**
 - OS name and version: Android 11
 - Manufacturer: Google
 - Model: Pixel 3

## Description

First of all, I'd like to thank you for such a great app! Even though I can live without pro version features, I decided to buy it as a form of support for the project.

### What you'd like to happen:

I'd like backlinks that are shown inside a note to be rendered as opposed to being shown as raw markdown.

Example:

**Other Note Title**
This is a sentence that [references](current-note.md) the current note.

as opposed to:

**Other Note Title**
```
This is a sentence that [references](current-note.md) the current note.
```

**Comment by hgad** [comment]
Similarly for search result snippets. I think it makes sense for those to be rendered too.

**Comment by hgad** [comment]
> Similarly for search result snippets. I think it makes sense for those to be rendered too.

Or at least to have an option to render them.

**Comment by vHanda** [comment]
Thank you for purchasing the Pro version. It really helps.

Till now I hadn't rendered the markdown everywhere because of performance reasons, however now that I've implemented proper Git log support (in dart_git), it allows me to easily cache the markdown results and start rendering it everywhere.

Rough Roadmap right now - Multi Repo Support, Obsidian Compatibility (there are lots of issues about this). You can expect this issue to be resolved in Nov (not a guarantee, but I would like to prioritize pro member requests)

**Reply by hgad** [reply]
Any updates on this issue?
